docs/application/toc_all.md

```diff
@@ -232,12 +219,37 @@
 #### [TEE Communication](/application/dotnet/guides/security/tee-client.md)
 #### [Device Policy Management](/application/dotnet/guides/security/dpm.md)
 
-### [Natural Language Processing](/application/dotnet/guides/nlp/nlp.md)
-
+### Sensors
+#### [Overview](/application/dotnet/guides/sensors/overview.md)
+#### [Device Sensors](/application/dotnet/guides/sensors/device-sensors.md)
+
+### System
+#### [Overview](/application/dotnet/guides/system/overview.md)
+#### [Attached Devices](/application/dotnet/guides/system/attached-devices.md)
+#### [System Information](/application/dotnet/guides/system/system.md)
+#### [System Settings](/application/dotnet/guides/system/settings.md)
+#### [Sound and Vibration Feedback](/application/dotnet/guides/system/feedback.md)
+
+###  Telephony
+#### [Overview](/application/dotnet/guides/telephony/index.md)
+#### [Telephony Information](/application/dotnet/guides/telephony/telephony.md)
+#### [Phone Number Management](/application/dotnet/guides/telephony/phonenumber.md)
+
+### UIX
+#### [Overview](/application/dotnet/guides/uix/overview.md)
```

Comment:
This file heading still reads Text Input and Voice

Reply:
Overview page title has been updated.

Reply:
@DariuszPaziewski 

-You have only changed the title, but the content is about text and voice, which makes the content irrelevant to the heading. 
- Please add a new page for UIX description rather and keep the current content under Text Input and Voice as separate. 

![image](https://user-images.githubusercontent.com/43839737/115206976-c9b06600-a118-11eb-9e33-047b4c04b83d.png)

Reply:
The previous page title ('Text Input and Voice') has been restored.